tmk_core/protocol/ps2_mouse.c

```diff
@@ -16,8 +16,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include <stdbool.h>
-#include <avr/io.h>
-#include <util/delay.h>
+
+#if defined(__AVR__)
```

Comment:
This include block should be removed and `#include "wait.h` used instead.

Reply:
+1 done